docs/source/workflow_inputs.rst

```diff
@@ -3431,6 +3431,7 @@ Each conventional storage water heater is entered as a ``/HPXML/Building/Buildin
   .. [#] TankModelType choices are "mixed" or "stratified".
   .. [#] NumberofBedroomsServed only required if IsSharedSystem is true.
          Tank losses will be apportioned to the dwelling unit using its number of bedrooms divided by the total number of bedrooms served by the water heating system per `ANSI/RESNET/ICC 301-2022 <https://codes.iccsafe.org/content/RESNET3012022P1>`_.
+         Zero bedroom dwelling units count as 1 bedroom.
```

Comment:
Adding this line for all `NumberofBedroomsServed` elements. This line implies that an entered value of zero is invalid (i.e., the shared system must serve at least 1 bedroom). Should we update the validator to reflect this? Seems like the current check on `NumberofBedroomsServed > NumberofBedrooms` covers it...

Reply:
I changed "Zero bedroom dwelling units count as 1 bedroom." to "The value entered here should count dwelling units w/zero bedrooms as 1 bedroom."